base/src/main/scala/co/uproot/abandon/Config.scala

```diff
@@ -217,14 +216,14 @@ case class NegativeConstraint(val accName: String) extends Constraint with SignC
   val signStr = "negative"
 }
 
-case class DateConstraint(dateFrom: Option[LocalDate], dateTo: Option[LocalDate]) extends Constraint {
+case class DateConstraint(dateFrom: Option[Date], dateTo: Option[Date]) extends Constraint {
   override def check(appState: AppState): Boolean = {
     appState.accState.posts.find(post => {
-      val from = dateFrom.getOrElse(LocalDate.MIN)
-      val to   = dateTo.getOrElse(LocalDate.MAX)
-      val date = LocalDate.of(post.date.year, post.date.month, post.date.day)
+      val from = dateFrom.getOrElse(Date.MIN)
+      val to   = dateTo.getOrElse(Date.MAX)
```

Comment:
While MIN and MAX works in most cases, it is hackish. It would be better to use something like this:

```
dateFrom.map(_ compare post.date).getOrElse(true)
```